ee/observe/explorer_handler_test.go

```
@@ -0,0 +1,629 @@
+func (s staticSchema) GetSchema(context.Context, string) (identity.Schema, error) {
+	return s.schema, nil
+}
+
+func serveExplorer(handler *ExplorerHandler, path string) *httptest.ResponseRecorder {
+	router := mux.NewRouter()
+	router.HandleFunc("/api/apps/{APP_ID}/observe/overview", handler.GetOverviewHandler)
+	router.HandleFunc("/api/apps/{APP_ID}/observe/events", handler.GetEventsHandler)
+	router.HandleFunc("/api/apps/{APP_ID}/observe/logs", handler.GetLogsHandler)
+	router.HandleFunc("/api/apps/{APP_ID}/observe/breakdown", handler.GetBreakdownHandler)
+	router.HandleFunc("/api/apps/{APP_ID}/observe/check-ins", handler.GetCheckInsHandler)
+	request := httptest.NewRequest(http.MethodGet, "/api/apps/"+uuid.NewString()+path, nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)